shell/edit/ml.llmos.ai.model.vue

```diff
@@ -56,10 +56,17 @@ export default {
   },
 
   data() {
-    const resource = { spec: { modelCard: { metadata: { datasets: [] } } } };
+    const resource = { 
+      spec: { 
+        registry: '',
+        modelCard: { metadata: { datasets: [] } },
+      },
+    };
 
     Object.assign(resource, this.value);
 
+    console.log('resource', resource);
```

Comment:
delete

Reply:
Fixed in https://github.com/llmos-ai/dashboard/pull/57